.changeset/six-pens-look.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+feat: Setup wizard updates
```

Comment:
can we have a better description of this "new feature"? something that would make sense for someone reading the release change log.